pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -127,16 +127,54 @@ message NamespaceOption {
     optional bool host_ipc = 3;
 }
 
+// PodSecurityContext holds pod-level security configuration that will be applied to a container.
```

Comment:
A little confused about PodSecurityContext vs SecurityContext.
Does PodSecurityContext functions as a default value when SecurityContext is not provided (in which case, the container level one overrides the pod level one).
or does PodSecurityContext overrides the SecurityContext??